app/assets/stylesheets/common/select-kit/email-group-user-chooser.scss

```diff
@@ -0,0 +1,39 @@
+.select-kit.email-group-user-chooser {
+  .select-kit-row.email-group-user-chooser-row {
+    .identifier {
+      color: var(--primary);
+      white-space: nowrap;
+    }
+    .name {
+      color: var(--primary-high);
+      font-size: $font-down-1;
+      margin-left: 0.5em;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+    }
+    .avatar,
+    .d-icon {
+      margin-left: 0;
+      margin-right: 0.5em;
+    }
+  }
+  .select-kit-header {
+    .x-more-item {
+      background: var(--primary-low);
+      padding: 0.25em;
+      flex: 1;
+      align-items: center;
+      display: flex;
+      justify-content: space-between;
+      box-sizing: border-box;
+      margin: 2px 0 0px 3px;
+      float: left;
+      height: 30px;
+      text-align: left;
+      color: inherit;
+      outline: none;
+      border-bottom: 2px solid transparent;
```

Comment:
Well, this CSS is for the '2 More' element that I wanted to have the same styling as the real choices so I simply stole their CSS properties and pasted them here 😅 The reason I didn't give it the same class is because select-kit would then think it's an actual choice since it queries for `.selected-name.choice` in a few places and I would have to exclude it.

![image](https://user-images.githubusercontent.com/17474474/106037336-9a74f480-60e7-11eb-8d40-7738b9157113.png)

That said, I reviewed the CSS properties here and discovered that the border was actually making the text go slightly off-center so I removed the border.